ethereum/contracts/BeefyLightClient.sol

```diff
@@ -93,32 +93,38 @@ contract BeefyLightClient {
     Blake2b public blake2b;
     uint256 public currentId;
     bytes32 public latestMMRRoot;
+    uint64 public latestBeefyBlock;
```

Comment:
```suggestion
    uint64 public latestPolkadotBlockNumber;
```

Reply:
Since Beefy has no blocks.

Reply:
well it refers to the latest block signed by beefy, so think it's still fine/makes sense. also this is a beefy light client not a polkadot light client so maybe better to keep it technically correct. anyway doesn't really matter.

Reply:
Fair point. A better name might be `latestRelayBlockNumber`, but agree, let's leave it as is ;)